tutormfe/templates/mfe/build/mfe/env.config.jsx

```diff
@@ -0,0 +1,51 @@
+{{- patch("mfe-env-config-buildtime-imports") }}
+
+function addPlugins(config, slot_name, plugins) {
+  if (slot_name in config.pluginSlots === false) {
```

Comment:
Is this idiomatic? Shouldn't it be `if(!(slot_name in config.pluginSlots))`? Or `if(config.pluginSlots[slot_name] === null)`?

Reply:
I think only `if (slot_name in config.pluginSlots)` is fine.

Reply:
On syntax: I guess it's a matter of opinion.  MDN [likes `=== false`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in#try_it) for negating the `in` operator.  But I do thing `in` is the best check, here.  It's unambiguous as to whether the property is defined or not.

But we do have to use the negation, here.  It avoids duplicating code.